### PR TITLE
Develop 160 cesm build

### PR DESCRIPTION
**Pull request by mattldawson**
updates to TUV-x related to #160, #162, and #164

- moves output of photo, dose rates to tuv-x stand alone driver
- adds `ENABLE_TESTS` and `ENABLE_UTIL_ONLY` build options to reduce build time when included in a 3D model
- removes calculation of diagnostics when diagnostic output is disabled

### CODE REVIEW

File: src/tuvx.F90 [hunk 1]
```diff
@@ -72,6 +73,206 @@ program tuvx
...
+      call core%run( sza%edge_val_( i_sza ),                                  &
+                     photolysis_rate_constants = photo_rates( i_sza, :, : ),  &
+                     dose_rates = dose_rates( i_sza, :, : ),                  &
+                     diagnostic_label = diagnostic_label )
+    end do
+
+    ! output results
+    if( core%number_of_photolysis_reactions( ) > 0 ) then
+      file_path = "photolysis_rate_constants.nc"
+      call output_photolysis_rate_constants( core, photo_rates, file_path )
```
Comment (K20shores) [inline src/tuvx.F90:115]: do we always need to be outputting these? If we're running in a host model won't we be transferring these rates in memory?

File: src/tuvx.F90 [hunk 2]
```diff
@@ -72,6 +73,206 @@ program tuvx
...
+    end if
+    if( core%number_of_dose_rates( ) > 0 ) then
+      file_path = "dose_rates.nc"
+      call output_dose_rates( core, dose_rates, file_path )
```
Comment (K20shores) [inline src/tuvx.F90:119]: same thing here

File: src/tuvx.F90 (hunk 1 above)
Comment (mattldawson) [reply src/tuvx.F90:115]: `src/tuvx.F90` is the driver for standalone TUV-x and isn't included in the TUV-x library. The 3D model will create its own instances of `core_t` from `src/core.F90`

File: src/tuvx.F90 (hunk 2 above)
Comment (mattldawson) [reply src/tuvx.F90:119]: (see above)

**Review by K20shores — APPROVED** [review]

**Pull request merged**